www/layouts/newsletter/section.html

```diff
@@ -0,0 +1,465 @@
+            <option value="Sweden">Sweden</option>
+            <option value="Switzerland">Switzerland</option>
+            <option value="Syria">Syria</option>
+            <option value="Taiwan">Taiwan</option>
+            <option value="Tajikistan">Tajikistan</option>
+            <option value="Tanzania">Tanzania</option>
+            <option value="Thailand">Thailand</option>
+            <option value="Timor-Leste">Timor-Leste</option>
+            <option value="Togo">Togo</option>
+            <option value="Tokelau">Tokelau</option>
+            <option value="Tonga">Tonga</option>
+            <option value="Trinidad and Tobago">Trinidad and Tobago</option>
+            <option value="Tunisia">Tunisia</option>
+            <option value="Turkey">Turkey</option>
+            <option value="Turkmenistan">Turkmenistan</option>
+            <option value="Turks &amp; Caicos Islands"
+              >Turks &amp; Caicos Islands</option
+            >
+            <option value="Turks and Caicos Islands"
+              >Turks and Caicos Islands</option
+            >
+            <option value="Tuvalu">Tuvalu</option>
+            <option value="Uganda">Uganda</option>
+            <option value="Ukraine">Ukraine</option>
+            <option value="United Arab Emirates">United Arab Emirates</option>
+            <option value="United Kingdom">United Kingdom</option>
+            <option value="Uruguay">Uruguay</option>
+            <option value="USA Minor Outlying Islands"
+              >USA Minor Outlying Islands</option
+            >
+            <option value="Uzbekistan">Uzbekistan</option>
+            <option value="Vanuatu">Vanuatu</option>
+            <option value="Vatican City State (Holy See)"
+              >Vatican City State (Holy See)</option
+            >
+            <option value="Venezuela">Venezuela</option>
+            <option value="Vietnam">Vietnam</option>
+            <option value="Virgin Islands (British)"
+              >Virgin Islands (British)</option
+            >
+            <option value="Virgin Islands (U.S.)">Virgin Islands (U.S.)</option>
+            <option value="Wallis and Futuna Islands"
+              >Wallis and Futuna Islands</option
+            >
+            <option value="Western Sahara">Western Sahara</option>
+            <option value="Yemen">Yemen</option>
+            <option value="Zambia">Zambia</option>
+            <option value="Zimbabwe">Zimbabwe</option>
+          </select>
+        </div>
+        <div id="mce-responses" class="clear">
+          <div
+            class="response"
+            id="mce-error-response"
+            style="display:none"
+          ></div>
+          <div
+            class="response"
+            id="mce-success-response"
+            style="display:none"
+          ></div>
+        </div>
+        <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
```

Comment:
I wonder what this comment and below div is about, maybe it is restricting bot signups like stated but I'm not sure.
if that's the case, then I wonder if we still require captcha

Reply:
This is a honeypot field to trick spam bots as explained
- https://www.smartfile.com/blog/captchas-dont-work-how-to-trick-spam-bots-with-a-smarter-honey-pot/
-https://mailchimp.com/help/about-fake-signups/

Reply:
Nice. Then I don't think we need to add captcha. 
Maybe @pdpinch can share his opinion on this.

Reply:
@Wassaf-Shahzad Let's resolve this conversation and you can discuss the captcha thing with Peter separately.